api/interservice/authz/v2/project.proto

```diff
@@ -72,13 +71,15 @@ service Projects {
 
     // "proper" methods go here (and the ones above will be phased out eventually)
     rpc CreateRule(CreateRuleReq) returns (CreateRuleResp) {};
+    rpc GetRule(GetRuleReq) returns (GetRuleResp) {};
+    rpc ListRules(ListRulesReq) returns (ListRulesResp) {};
```

Comment:
What about `ListProjectRules` and `GetProjectRules` above? Are these replacing those?

Reply:
No those two endpoints are [slightly different](https://github.com/chef/automate/blob/master/components/authz-service/projects_spike.md#endpoints) and will be updated when we get to [this card](https://chefio.atlassian.net/browse/A2-809).